vmdb/app/models/miq_report/generator/sorting.rb

```diff
@@ -49,7 +49,6 @@ def build_sort_suffix_data
     rpt_sortby = self.sortby.to_miq_a
     rpt_sortby.each do |sb|
       col, sfx = sb.split("__")
-      dt = self.class.get_col_type(self.col_to_expression_col(sb))
```

Comment:
OH sort by!  Man, I hate abbreviations...I mean we're not writing old Fortran here. :stuck_out_tongue_closed_eyes: